CHANGELOG.md

```diff
@@ -18,6 +18,15 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
  - rename `rtic` feature to `rtic1`, add `rtic2` [#727]
  - rename `stm32_i2s_v12x` feature to `i2s` [#718]
  - Check features enabled in `build-rs` [#718]
+  - add trait bound `RegisterBlockImpl` to type `RegisterBlock` associated with `serial::Instance` [#732]
+  - removed unneeded trait bound for methods that take in a `serial::Instance` and use the associated `RegisterBlock`
+
+### Changed
+
+ - complete and rework Dma Stream API [#666]
+ - SPI bidi takes 2 pins [#526]
+ - `Fast Read Quad I/O (EBh)` in `qspi-w25q` example now matches W25QXX datasheet. [#682]
+ - `embedded-storage` version bumped to 0.3
```

Comment:
move changelog in `Unreleased` section, please. And remove unrelated changes.

Reply:
and squash commits

Reply:
These should both be fixed in the lastest commit. Not sure how I messed up the changelog. Thanks for catching that.